marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.tsx

```diff
@@ -458,9 +459,9 @@ const AddEditRelationship: React.FC<Props> = (props) => {
     //menuProps.unshift({value: "None", label: "None", type: "string"});
     if (menuProps) {
       if (!isConceptClassJoinView) {
-        menuProps.unshift({value: "None", label: "None", type: "string"});
+        menuProps.unshift({value: "None", key: "None", label: "None", type: "string"});
       } else {
-        menuProps.unshift({value: ".", label: "instance", type: "string"});
+        // menuProps.unshift({ value: ".", key:".", label: " - instance - ", type: "string" });
```

Comment:
there's a comment here, should be removed but is it there for a reason or something?